packages/util/src/string.ts

```diff
@@ -1,3 +1,53 @@
+/**
+ * Encode a string to a base64 encoded string.
+ * @param str String to encode.
+ * @returns Base64 encoded string.
+ */
 export function encodeBase64(str: string): string {
   return Buffer.from(str).toString('base64');
 }
+
+/**
+ * Prepend a string by the given indentation.
+ * @param string String to indent.
+ * @param indentation Indentation to use.
+ * @returns A string with a prefix consisting of spaces with the length of the given indentation.
+ */
+export const indent = (string, indentation) =>
+  string
+    .split('\n')
+    .map(subString => `${indentation}${subString}`)
+    .join('\n');
+
+/**
+ * Remove whitespace from the left side of a string.
+ * @param string String to trim.
+ * @returns String without whitespace on the left side.
+ */
+export const trimLeft = string => {
```

Comment:
General: Function definition via the arrows is a matter of taste. I like the funciton keyword. But any way you define it we should add stronger typing: Argument of type string and methods returns a string.

Reply:
Will do

Reply:
Done.